packages/acgtk/acgtk.2.0.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+name: "acgtk"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+install: ["dune" "install"]
+
+depends: [
+  "ocaml" { >= "4.13.0" }
+  "dune" { >= "3.5" }
+  "menhir" { >= "20211230"}
+  "ocamlgraph"
+  "ANSITerminal" { >= "0.8" }
+  "fmt"
+  "logs"
+  "mtime" { >= "2.0.0"}
+  "cmdliner" { >= "1.1.0"}
+  "sedlex"
+  "cairo2"
+  "yojson" { >= "1.6.0"}
+  "readline"
+  "ocamlformat" { dev }
```

Comment:
```suggestion
```
the `dev` is [probably what you meant for](https://opam.ocaml.org/doc/Manual.html#pkgvar-dev). before the arrival of opam.2.2 (currently in alpha) there is no way to specify development dependencies (see also https://github.com/ocaml/opam/issues/4959)